src/app.ts

```diff
@@ -21,6 +22,11 @@ const corsOptions = {
   },
 };
 
+const pwRequestLimiter = rateLimit({
+  windowMs: 10 * 60 * 1000,
```

Comment:
There are helper constants for time that I pretty much always add to the `constants` file in every project. In this case use `10 * MINUTE_MILLIS`. This way it's much easier to immediately understand what's going on